docs/contributing/code_of_conduct.rst

```diff
@@ -1,4 +1,165 @@
-.. include:: ../../../CODE_OF_CONDUCT.rst
+..
+  This file is also maintained as CODE_OF_CONDUCT.md for Github. You can use a
+  tool like `pandoc --atx-headers -o CODE_OF_CONDUCT.md docs/contributing/code_of_conduct.rst`
+  to convert this file to Markdown.
```

Comment:
Rather than maintaining two versions in parallel, would be cleaner to have a single source of truth. possible to import or otherwise reference the .md version?

https://docs.readthedocs.io/en/stable/intro/getting-started-with-sphinx.html#using-markdown-with-sphinx

Reply:
I agree in principle, but we never changed anything in the CoC, so it's not important.. and the DRY ideal was what broke this setup initially (besides the non-failing Sphinx build).

Reply:
(It's copied by running a single `pandoc` command which is listed in that comment)

Reply:
> the DRY ideal was what broke this setup initially

I actually still don't understand that part of #4809 - the integration is still working as far as I can tell. When I navigate to our current developer docs, I can easily access the code of conduct:

1. ![image](https://user-images.githubusercontent.com/2367265/51935253-58aaf980-23ba-11e9-9226-b7e9e12a641a.png)
2. ![image](https://user-images.githubusercontent.com/2367265/51935277-652f5200-23ba-11e9-949f-a8aba40fe577.png)
3. ![image](https://user-images.githubusercontent.com/2367265/51935297-724c4100-23ba-11e9-8967-3535c7b30656.png)

It's just missing from github. Am I missing something?

Reply:
1) Radina asked this same question so don't feel bad about asking it, too :)
2) I posted the screenshot in the issue
3) The whole Code of Conduct is missing :wink:

Reply:
Also the Code of Conduct is missing on Github, too. And that's also stated in the issue.